### ISSUE TITLE
Allow skipping entries

### ISSUE THREAD

**Issue by skx** [skx/rss2email#58]
I'd like to be able to skip entries from particular feeds, via a regular expression match on the title and body.

This must be per-feed, rather than global.

**Comment by skx** [comment]
This is blocked by the configuration-file, because it requires a per-feed list of patterns to skip.

**Comment by skx** [comment]
Closed in #62.

**Comment by woulf** [comment]
Hello, very nice project!
Would it be possible to do the opposite: ie only include some regex?

**Comment by skx** [comment]
That's not a bad idea, please do [open a new issue](https://github.com/skx/rss2email/issues/new), and I'll look into it over the next couple of days.
